modules/core/keeper/msg_server.go

```diff
@@ -279,6 +282,13 @@ func (k Keeper) ChannelOpenInit(goCtx context.Context, msg *channeltypes.MsgChan
 		return nil, sdkerrors.Wrap(err, "channel open init callback failed")
 	}
 
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			sdk.EventTypeMessage,
+			sdk.NewAttribute(sdk.AttributeKeyModule, connectiontypes.AttributeValueCategory),
+		),
+	})
```

Comment:
what was the purpose of this event? Does this work with hermes? I think it causes duplicate event emission and could lead to incorrect event parsing. This event is emitted in [baseapp](https://github.com/cosmos/cosmos-sdk/blob/8297f2f3125e6b729b4377144246b38f8b73b7a1/baseapp/baseapp.go#L744) per each SDK message received.